ephios/core/templates/core/disposition/fragment_participation.html

```diff
@@ -43,6 +46,9 @@ <h6 class="d-inline">
             <button class="btn" type="button" data-bs-toggle="collapse"
                     data-bs-target="#participation-collapse-{{ form.instance.id }}" aria-expanded="false"
                     aria-controls="collapseExample">
+                {% if additional_html %}
+                    <span class="fas fa-info-circle pe-2"></span>
```

Comment:
I'm not sure if it is smart to do highlighting just if additional html is present.
- There may be a usecase where a plugin wants to render HTML for every participation which is not critical for disposition, but nice-to-have context
- When a question is required, every participation will have this indicator.

Both could lead to a visual overload for the user without providing a benefit/additional information

Reply:
hmmm, difficult decision. I think I am leaning a bit towards not showing anything, as it might be a question where only the aggregation is relevant (e.g. food) or the planners will remember that they specifically added a question that they want to see the answer of.

@felixrindt what is your take on this?

Reply:
I'd go with no icon, as in many situations noone or everyone would get one. Let's see how people use the questions, first.